wingtips-zipkin2/src/main/java/com/nike/wingtips/zipkin2/util/WingtipsToZipkinSpanConverterDefaultImpl.java

```diff
@@ -60,4 +64,53 @@ else if (SpanPurpose.LOCAL_ONLY == wtsp || SpanPurpose.UNKNOWN == wtsp) {
             return null;
         }
     }
+
+    protected String sanitizeIdIfNecessary(final String originalId, final boolean allow128Bit) {
```

Comment:
There's no log correlation happening here - no way to easily know that a given bad ID was sanitized (and if it was, what it was sanitized to).

For any sanitization I recommend a log message along the lines of `logger.info("Detected invalid ID format. orig_id={}, sanitized_id={}", originalId, sanitizedId)`. That way when you search the logs for a given invalid ID, you'll see this message telling you what the sanitized ID is so you can go look it up in zipkin/etc.